index.js

```diff
@@ -23,16 +29,23 @@ function listObjectInner() {
 S3Zipper.prototype = {
     init: function (awsConfig) {
         this.awsConfig = awsConfig;
-        AWS.config.update({
-            accessKeyId: awsConfig.accessKeyId,
-            secretAccessKey: awsConfig.secretAccessKey,
-            region: awsConfig.region
-        });
-        this.s3bucket = new AWS.S3({
-            params: {
-                Bucket: this.awsConfig.bucket
+        AWS.config.getCredentials(function (err) {
```

Comment:
@DanielHindi when using aws-s3-zipper on AWS EC2 the credentials are already configured, or if you have configured AWS previously or using environment variables there is no need to configure it again.